doc/rados/operations/crush-map-edits.rst

```diff
@@ -629,29 +641,38 @@ There are three types of transformations possible:
    single bucket.  For example, in the previous example, we want the
    ``ssd`` bucket to be mapped to the ``default`` bucket.
 
-The final command to convert the map comprised of the above fragments would be something like::
+The final command to convert the map comprised of the above fragments would be something like:
```

Comment:
s/comprised of/comprising/

Reply:
My sense is that both are acceptable.  "comprise" is so protean and slippery that I personally usually find a different phrasing to avoid being stilted. ymmv.

Reply:
I have fixed this in https://github.com/ceph/ceph/pull/48362.